apps/paper/pages/programmable-intents/operators.mdx

```diff
@@ -33,6 +33,13 @@ In the realm of cross-chain transactions, the LL introduces an innovative and ef
 
 Should the final transaction cost fall below the user-specified maximum fee, the LL guarantees the reimbursement of the residual amount to the user. This approach not only fosters a competitive environment for transaction routing but also ensures transparency and fairness in transaction costs, ultimately benefiting the end users by providing them with cost-effective and timely transaction services. Through these meticulously designed operational roles and mechanisms, the LL exemplifies a forward-thinking approach to blockchain infrastructure management, emphasizing security, efficiency, and user-centricity.
 
+### Cross-Chain Trilemma
+
+![Cross-Chain Trilemma](/cross-chain-trilemma.png)
+
+The concept of the cross-chain trilemma highlights a challenge in cross-chain infrastructure design, where only two out of three desirable attributes—execution guarantee, low fees, and fast execution—can be achieved simultaneously. This dilemma arises due to design decisions that influence the trade-offs between these attributes. In-protocol paths enable cheaper transfers by using predefined routes across chains, sacrificing speed to mitigate reorganization risks without needing extra validators or incurring liquidity costs. Solver aggregation aims to find the most cost-effective and swift transaction path by comparing multiple solvers, but issues like adverse selection and front-running can compromise execution reliability. Execution competition, where solvers vie to fulfill transactions either through direct competition or exclusive selection, tends to drive up user fees as the focus shifts from reducing costs to winning execution rights.
```

Comment:
I'm pretty sure we should cite https://frontier.tech/ here, as they have invented the term as well as the image of the diagram.

Maybe deserves a section of why we're doing this and how we're approaching chain abstraction

Reply:
agreed with the citation. Do you have an example (on another page) where you did citation? I will follow the same format. 

If not, I will just add a link and names of the authors

Reply:
Yes - i do not have it yet 😅